charts/app-config/values-integration.yaml

```diff
@@ -1701,17 +1701,32 @@ govukApplications:
 - name: static
   helmValues:
     extraEnv:
+      - name: GA_UNIVERSAL_ID  # Not a secret.
+        value: &ga-universal-id UA-26179049-22
+      - name: GOOGLE_TAG_MANAGER_ID  # Not a secret.
+        value: &static-gtm-id GTM-MG7HG5W
+      - name: GOOGLE_TAG_MANAGER_AUTH  # Non-prod only.
```

Comment:
Sounds like perhaps this doesn't need to be a secret after all: https://github.com/alphagov/govuk-puppet/pull/8041

I couldn't find anything on the web about what knowing a `gtm_auth` value might allow someone to do, so I've asked UX measurement team.

Reply:
Verdict from UX measurement team: not secret 👍

Summary of my understanding of this now: if someone wanted to push fake data to our GA account (which they can do anyway), knowing this value would let them have that data be processed with the GA/GTM configuration that's being previewed on the integration/staging site as opposed to the configuration for the live site. In other words, it's useless for nefarious purposes.